eclass/kernel-build.eclass

```diff
@@ -114,6 +114,15 @@ kernel-build_pkg_setup() {
 	python-any-r1_pkg_setup
 	if [[ ${KERNEL_IUSE_MODULES_SIGN} ]]; then
 		secureboot_pkg_setup
+		if [[ -e ${MODULES_SIGN_KEY} && -e ${MODULES_SIGN_CERT} &&
+			${MODULES_SIGN_KEY} != ${MODULES_SIGN_CERT} &&
```

Comment:
This copying is only done when `MODULES_SIGN_KEY` and `MODULES_SIGN_CERT` are separate files (the kernel expects both cert and key in the same pem file). If we have a permission problem, then we should probably also do this copy/chmod/chown if we do provide the cert and key in the same file (i.e. MODULES_SIGN_KEY is set, but MODULES_SIGN_CERT is not, or they are set to the same value).

Reply:
Yep, realized that a couple days ago, haven't gotten around to fixing it yet. I'll try to do that today.